htdocs/core/class/commondocgenerator.class.php

```diff
@@ -145,7 +145,11 @@ public function get_substitutionarray_user($user, $outputlangs)
 			'myuser_email'=>$user->email,
 			'myuser_logo'=>$logotouse,
 			'myuser_job'=>$user->job,
-			'myuser_web'=>''	// url not exist in $user object
+			'myuser_web'=>''	// url not exist in $user object,
+			'myuser_birth'=>dol_print_date($user->birth, 'day', 'tzuser'),
```

Comment:
A birthday must not differ with the position of the user on earth. 
So here i think you must use
'myuser_birth'=>dol_print_date($user->birth, 'day', 'tzserver'),
or
'myuser_birth'=>dol_print_date($user->birth, 'day', 'gmt'),

I don't remember if we store the birthday as tzserver date or gmt date.